OverlayPlugin.Core/MemoryProcessors/AtkStage/FFXIVClientStructs/Extensions/FFXIV/Client/Game/Object/GameObject.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace FFXIVClientStructs.Global.FFXIV.Client.Game.Object
+{
+    // Size = 0x1A0
+    public unsafe partial struct GameObject
+    {
+        // `Name` already present, size is 4 bytes smaller (64 bytes)
```

Comment:
I think I just didn't know how big the name was.  I don't know that the "4 bytes smaller" comment is needed necessarily, since the 68 bytes size is removed elsewhere.

Reply:
I think that on the extremely unlikely edge case of a name actually being a full 64 bytes long, that could overflow into the value afterwards instead of getting cut off due to the null bytes that the name field is filled with otherwise.

So maybe this should be fixed in a different PR instead, for all game versions, to prevent that edge case?

Reply:
I think it's unclear if such an edge case exists.  It really depends on how big of a string the game itself wants to write into that space.  It's possible that the name field is even shorter than that, or that it always fits the null terminator into byte 64; I think it'd depend on the code.

Mostly just saying that I have no idea how big it is.  I wrote 68 because that's where the next field I knew about was.  I don't know that 64 is necessarily right either (although certainly more plausible).  Matching what FFXIVClientStructs is doing seems fine, but mostly saying I don't know if this "fixes" anything.